packages/flutter/test/material/elevated_button_test.dart

```diff
@@ -10,6 +10,13 @@ import 'package:flutter_test/flutter_test.dart';
 import '../widgets/semantics_tester.dart';
 
 void main() {
+  TextStyle iconStyle(WidgetTester tester, IconData icon) {
+    final RichText iconRichText = tester.widget<RichText>(
+      find.descendant(of: find.byIcon(icon), matching: find.byType(RichText)),
+    );
+    return iconRichText.text.style!;
+  }
```

Comment:
As far as "global functions" vs. "functions inside main", I'm not sure if one is better than the other. The style guide has a section called [**Make each test entirely self-contained**](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#make-each-test-entirely-self-contained), but the goal of the rule is to avoid shared state, so a helper function like this one should be good 👍

Reply:
Generally we put functions inside main.